app/src/main/java/com/mybrain/playlistmaker/client/ItunesApi.kt

```diff
@@ -0,0 +1,14 @@
+package com.mybrain.playlistmaker.client
+
+import com.mybrain.playlistmaker.dto.ItunesSearchResponseDto
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface ItunesApi {
+    @GET("search")
```

Comment:
entity можно прям сюда перенести:
```suggestion
    @GET("search?entity=song")
```